### PR TITLE
Removed unused variables

### PR DESCRIPTION
**Pull request by nc-kano**

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.Droid/Views/Settings/ConsentSettingPageFragment.cs [hunk 1]
```diff
@@ -29,7 +29,6 @@ public override View OnCreateView(LayoutInflater inflater, ViewGroup container,
             View view = inflater.Inflate(Resource.Layout.consent_settings_page_body, container, false);
 
             RelativeLayout RelativeLayout4 = view.FindViewById<RelativeLayout>(Resource.Id.consent_paragraph_hvordan_accepterer);
-            TextView consentFourParagraph = RelativeLayout4.FindViewById<TextView>(Resource.Id.consent_page_title);
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.Droid/Views/Settings/ConsentSettingPageFragment.cs:32]: Have you tested that this is not needed?

File: NDB.Covid19/NDB.Covid19.Droid/Views/Welcome/WelcomePageConsentsActivity.cs [hunk 2]
```diff
@@ -99,7 +99,6 @@ protected override void OnCreate(Bundle savedInstanceState)
 
 
             RelativeLayout RelativeLayout4 = FindViewById<RelativeLayout>(Resource.Id.consent_paragraph_hvordan_accepterer);
-            TextView consentFourTitle = RelativeLayout4.FindViewById<TextView>(Resource.Id.consent_page_title);
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.Droid/Views/Welcome/WelcomePageConsentsActivity.cs:102]: Have you tested that this is not needed?

File: NDB.Covid19/NDB.Covid19.Droid/Views/Settings/ConsentSettingPageFragment.cs (hunk 1 above)
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.Droid/Views/Settings/ConsentSettingPageFragment.cs:32]: yes, IDE even marks an error here as Resource.Id.consent_page_title does not exist anymore

Comment (Frederik-S-N-dk) [reply NDB.Covid19/NDB.Covid19.Droid/Views/Settings/ConsentSettingPageFragment.cs:32]: Hmmm... exciting... I didn't show any error when I implemented it :)

Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.Droid/Views/Settings/ConsentSettingPageFragment.cs:32]: wrong IDE ;) i am using Rider

**Review by Frederik-S-N-dk — APPROVED** [review]

**Pull request merged**